src/chttpd/src/chttpd_util.erl

```diff
@@ -95,3 +99,68 @@ handle_legacy_config(OriginalHeaders, DefaultHeaderValue) ->
         false ->
             OriginalHeaders
     end.
+
+handle_request_helper(Req) ->
```

Comment:
I think we may want a better name here. `handle_request_helper` seems rather generic? It would imply it can handle any request, but it can't.

Reply:
Change to `handle_all_dbs_or_dbs_info_req_helper`